### PR TITLE
Add build script for 3scale/apisonator

### PR DESCRIPTION
**Pull request by krishvoor**
Adds built scripts for 3scale/apisonator

Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

### CODE REVIEW

File: 3/3scale-apisonator/redhat_ubi8/build.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+(git clone $BRANCH https://github.com/3scale/apisonator) || (echo "git clone failed"; exit $?)
+cd apisonator
+
+# build
+make ci-build
+
+# run tests
+make DOCKER_OPTS="-e TEST_ALL_RUBIES=1" test
+
+
```
Comment (Abhijit-Mane) [inline 3/3scale-apisonator/redhat_ubi8/build.sh:38]: No copying of build artifacts needed? If not, then it's fine.

Comment (krishvoor) [reply 3/3scale-apisonator/redhat_ubi8/build.sh:38]: We are building the container images, no artefacts as such.

**Comment by Abhijit-Mane** [comment]
I see you are using a build script directly. Are we not using the Dockerfile approach?

**Review by Abhijit-Mane — APPROVED** [review]
LGTM

**Pull request merged**